xapian-applications/omega/templates/query

```diff
@@ -1,24 +1,24 @@
-$httpheader{Content-Type,text/html; charset=utf-8}<html>
+$httpheader{Content-Type,text/html; charset=utf-8}<!DOCTYPE html><html lang="en">
 $set{flag_spelling,$ne{$cgi{SPELL},0}}
 $set{thousand,$.}$set{decimal,.}
 $setmap{prefix,,$split{ S B F},title,S,author,A,topic,B}
 ${
 $def{PREV,
 $if{$ne{$topdoc,0},<input type=image name="&lt;" alt="&lt;"
-src="/icons/omega/prev.png" border=0 height=30 width=30>,
+src="/icons/omega/prev.png" style="border:0px;" height=30 width=30>,
```

Comment:
Although we don't have a standard for this, I'd like a space after the colon in inline CSS please :-)

(Better yet actually might be to put it in a `<style>` block, since it's for all `input[type=image]`.